spec/support/api/schemas/artists.json

```diff
@@ -25,15 +25,27 @@
           "spotify_uri": {
             "id": "http://spotify-apprentice-app.com/artists/0/spotify_uri",
             "type": "string"
+          },
+          "song_ids": {
+            "id": "http://spotify-apprentice-app.com/artists/0/song_ids",
+            "type": "array",
+            "items": {
+              "id": "http://spotify-apprentice-app.com/artists/0/song_ids/0",
+              "type": "string"
+            }
           }
         },
         "required": [
           "id",
           "name",
           "description",
-          "spotify_uri"
+          "spotify_uri",
+          "song_ids"
         ]
-      }
+      },
+      "required": [
+        "0"
+      ]
```

Comment:
You actually don't want this required key or it'll insist that there's at least one artist in the array